catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java

```diff
@@ -97,6 +97,7 @@
 	private FormulaBrick clonedFormulaBrick;
 	private Brick.BrickField currentBrickField;
 	private Formula currentFormula;
+	private Menu currentMenu;
 
```

Comment:
I did not expand the `github-diff (@@ -483,6 +486,14 @@)`, so it looked like the `currentMenu` variable gets used in `onPrepareOptionsMenu`.